### PR TITLE
Create your own Enumerable

### PR DESCRIPTION
**Pull request by NasKhalil**
This branch includes the following featured implemented:

- Create a class MyList that has an instance variable @list.
- Create a module MyEnumerable that implements the following methods:
- - #all?
- - #any?
- - #filter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ To get a local copy up and running follow these simple example steps.
 Clone the project by using terminal:
 
 ```
-git clone https://github.com/NasKhalil/Morse-code.git
+git clone https://github.com/ReAliens/enumerable.git
 
 cd Morse-code
```
Comment (julie-ify) [inline README.md]: - [x] Kindly remove everything that has to do with `morse-code` and let's make sure we `cd` to this current project and how to run this current project 😉

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @NasKhalil and @ReAliens,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (NasKhalil) [reply README.md]: Hi @julie-ify 

Thanks for your time spent reviewing our work, it was a lack of concentration.

Cheers 👋

Comment (ReAliens) [reply README.md]: thanks i will implement this now :sweat_smile:

**Review by Lameck1 — APPROVED** [review]
Hi, @NasKhalil & @ReAliens,

Tasks for this project have been successfully implemented 💯.
You've done well working on the previous review concerns 👏🏿.

# Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (julie-ify) [reply README.md]: You guys rock 🔥

**Comment by ReAliens** [comment]
thanks, for your time :smile:

**Pull request merged**